prov/util/src/util_ep.c

```diff
@@ -311,6 +333,11 @@ int ofi_endpoint_close(struct util_ep *util_ep)
 		ofi_atomic_dec32(&util_ep->av->ref);
 	}
 
+	if(util_ep->coll_cid_mask) {
```

Comment:
one of my next goals is to have you write if statements with a space between the if and (
:)